### PR TITLE
Updates to CircleCI configuration

### PR DESCRIPTION
**Pull request by kevinlind**
## Description

Updates to CircleCI configuration to get CI builds to pass.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by emdobrin** [comment]
@kevinlind nice, the published artifacts are super helpful 🎉 
Do you know if we can filter them out so they only show the main index file for each type of report e.g. public javadocs, coverage, etc and hide the rest?

File: .circleci/config.yml [hunk 1]
```diff
@@ -145,34 +149,34 @@ jobs:
...
           cache-prefix: << parameters.build-cache-prefix >>
 
+      - run:
+          name: List availabe emulator images
+          command: sdkmanager --list
+
       - android/start-emulator-and-run-tests:
           #   It should match the name seen in the "sdkmanager --list" output
-          system-image: system-images;android-33;google_apis;x86_64
+          system-image: system-images;android-29;default;x86
```
Comment (kevinlind) [inline .circleci/config.yml:161]: I'm keeping the same system-image we are using for the other Android Edge extensions. This [CircleCI blog post on the Android Emulator](https://support.circleci.com/hc/en-us/articles/360000028928-Testing-with-the-Android-emulator-on-CircleCI) states that test issues have been observed with the higher API level images which use Google APIs and it is suggested to use lower API images.

File: .circleci/config.yml [hunk 2]
```diff
@@ -145,34 +149,34 @@ jobs:
...
       - checkout
 
-      - android/restore-gradle-cache:
-          cache-prefix: << parameters.gradle-cache-prefix >>
       - android/restore-build-cache:
           cache-prefix: << parameters.build-cache-prefix >>
 
+      - run:
+          name: List availabe emulator images
+          command: sdkmanager --list
```
[the remark this answers is not in the document]
Comment (emdobrin) [reply .circleci/config.yml:157]: ok, just a small typo "List available ..."

File: .circleci/config.yml [hunk 3]
```diff
@@ -145,34 +149,34 @@ jobs:
...
+          file: ci/functional-test/build/reports/coverage/androidTest/phone/debug/connected/report.xml
           flags: functional-tests
-          validate: true
 
       - store_artifacts:
           path: ci/functional-test/build/reports
 
       - store_test_results:
-          path: ci/functional-test/build/outputs/androidTest-results
+          path: code/edge/build/outputs/androidTest-results
```
[the remark this answers is not in the document]
Comment (emdobrin) [reply .circleci/config.yml:181]: can you add a short comment on this  in the config file as a reminder for maintainers?

**Review by emdobrin — APPROVED** [review]

**Pull request merged**